### PR TITLE
Add some documentation for the Spark module

### PR DESCRIPTION
**Pull request by sbrunk**

### CODE REVIEW

File: docs/pages/usage-spark.md [hunk 1]
```diff
@@ -1,3 +1,102 @@
 ---
 title: Spark
----
\ No newline at end of file
+---
+
+Almond comes with a Spark integration module called *almond-spark*, which allows you to connect to a Spark cluster and
+to run Spark calculation interactively from a Jupyter notebook.
```
Comment (alexarchambault) [inline docs/pages/usage-spark.md]: s/calculation/calculations/
```suggestion
to run Spark calculations interactively from a Jupyter notebook.
```

File: docs/pages/usage-spark.md [hunk 2]
```diff
@@ -1,3 +1,102 @@
...
+
+```scala
+def sc = spark.sparkContext
+
+val rdd = sc.parallelize(1 to 100000000, 100)
+
+val n = rdd.map(_ + 1).sum()
+```
+
+When you execute a Spark `Action` like run you should see a progress bar, showing the progress of the running Spark job,
```
Comment (alexarchambault) [inline docs/pages/usage-spark.md]: shouldn't it be like this?
```suggestion
When you execute a Spark action like `sum` you should see a progress bar, showing the progress of the running Spark job,
```

File: docs/pages/usage-spark.md [hunk 3]
```diff
@@ -1,3 +1,102 @@
...
+as well as a link to cancel the job.
```
Comment (alexarchambault) [inline docs/pages/usage-spark.md]: For now, the kill button needs some Javascript code to run from the front-end, to send a comm message to the kernel. So it only works from the Jupyter classic UI (jupyterlab doesn't run Javascript out-of-the-box).

I guess it shouldn't be shown by default (and only some Javascript code would make it visible, ensuring some Javascript code can actually be run).

```suggestion
as well as a link to cancel the job if you are using the Jupyter classic UI.
```

File: docs/pages/usage-spark.md (hunk 2 above)
Comment (sbrunk) [reply docs/pages/usage-spark.md]: Yeah, it was late... Thanks for catching.

**Comment by alexarchambault** [comment]
Merging then, thanks!

**Pull request merged**